From 7618498eec840cb26474dc47821e083957772706 Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Fri, 20 Mar 2009 14:37:24 +1000
Subject: [PATCH] Add x11_driver for American Megatrends KVM.

This device exposes only ABS_X, ABS_Y and buttons. It gets input.joystick
assigned but not input.mouse, hence the default evdev mapping doesn't pick up
on it.

Red Hat Bug 484776 <https://bugzilla.redhat.com/show_bug.cgi?id=484776>

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---
 fdi/policy/10osvendor/10-x11-input.fdi |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/fdi/policy/10osvendor/10-x11-input.fdi b/fdi/policy/10osvendor/10-x11-input.fdi
index 769f75b..eb9767e 100644
--- a/fdi/policy/10osvendor/10-x11-input.fdi
+++ b/fdi/policy/10osvendor/10-x11-input.fdi
@@ -2,6 +2,12 @@
 <deviceinfo version="0.2">
   <device>
 
+    <!-- This KVMS has abs x/y and buttons. It only has inputInfo.joystick
+         set by hal and doesn't get picked up by default -->
+    <match key="input.product" contains="American Megatrends Inc. Virtual Keyboard and Mouse">
+       <merge key="input.x11_driver" type="string">evdev</merge>
+    </match>
+
     <!-- KVM emulates a USB graphics tablet which works in absolute coordinate mode -->
     <match key="input.product" contains="QEMU USB Tablet">
        <merge key="input.x11_driver" type="string">evdev</merge>
-- 
1.6.0.6

